src/main/java/com/minecolonies/coremod/util/constants/RenderUtils.java

```diff
@@ -0,0 +1,144 @@
+package com.minecolonies.coremod.util.constants;
+
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.ColonyView;
+import com.minecolonies.coremod.configuration.Configurations;
+import com.minecolonies.coremod.util.BlockPosUtil;
+import com.minecolonies.structures.helpers.Settings;
+import com.minecolonies.structures.helpers.Structure;
+import com.minecolonies.structures.lib.ModelHolder;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.entity.EntityPlayerSP;
+import net.minecraft.client.multiplayer.WorldClient;
+import net.minecraft.client.renderer.block.model.IBakedModel;
+import net.minecraft.entity.item.EntityEnderCrystal;
+import net.minecraft.init.Blocks;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * I need that to calculate the circle, if someone knows a better library tell me.
+ */
+import static sun.awt.geom.Curve.next;
```

Comment:
@Raycoms 

> Classes in the sun.* or com.sun.* packages are considered implementation details, and are not part of the Java API.
> 
> They can cause problems when moving to new versions of Java because there is no backwards compatibility guarantee. Similarly, they can cause problems when moving to a different Java vendor, such as OpenJDK.
> 
> Such classes are almost always wrapped by Java API classes that should be used instead.
>